### PR TITLE
[Changelog] Add SE-0202 to 4.2

### PR DESCRIPTION
**Pull request by Azoy**

### CODE REVIEW

**Comment by Azoy** [comment]
cc: @natecook1000

**Comment by harlanhaskins** [comment]
ping: @natecook1000

File: CHANGELOG.md [hunk 1]
```diff
@@ -74,6 +74,51 @@ Swift 4.2
...
+  ```swift
+  let greetings = ["hey", "hello", "hi", "hola"]
+  let randomGreeting = greetings.randomElement()! // This returns an Optional
+  let newGreetings = greetings.shuffled() // ["hola", "hi", "hey", "hello"]
+  ```
+
+  Core to the randomization functionality is a new `RandomNumberGenerator`
+  protocol. The standard library defines its own random number generator
+  called `SystemRandomNumberGenerator` which is backed by a secure and thread
+  safe random number generator on each platform. All the randomization functions
```
Comment (natecook1000) [inline CHANGELOG.md]: Can you hyphenate "thread-safe"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -74,6 +74,51 @@ Swift 4.2
...
+  
+  var mt = MersenneTwister()
+  let diceRoll = Int.random(in: 1 ... 6, using: &mt)
+  let randomUnit = Double.random(in: 0 ..< 1, using: &mt)
+  let randomBool = Bool.random(using: &mt)
+  
+  // Collection additions
+  let greetings = ["hey", "hello", "hi", "hola"]
+  let randomGreeting = greetings.randomElement(using: &mt)! // This returns an Optional
+  let newGreetings = greetings.shuffled(using: &mt) // ["hola", "hi", "hey", "hello"]
```
Comment (natecook1000) [inline CHANGELOG.md]: I don't think we need to repeat all these here — maybe just one example of using `mt`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -74,6 +74,51 @@ Swift 4.2
...
+  protocol. The standard library defines its own random number generator
+  called `SystemRandomNumberGenerator` which is backed by a secure and thread
+  safe random number generator on each platform. All the randomization functions
+  have a `using:` parameter that take a `RandomNumberGenerator` that users can
+  pass in their own random number generator.
+  
+  ```swift
+  struct MersenneTwister: RandomNumberGenerator {
+    func next() -> UInt64 {
+      // Algorithm stuff here
```
Comment (natecook1000) [inline CHANGELOG.md]: `// implementation` instead?

File: CHANGELOG.md [hunk 4]
```diff
@@ -74,6 +74,51 @@ Swift 4.2
 
 ### 2018-09-17 (Xcode 10.0)
 
+* [SE-0202][]
+
+  The standard library now provides a unified set of randomization functionality.
+  Integer types, floating point types, and Bool all introduce a new random
+  function.
```
Comment (natecook1000) [inline CHANGELOG.md]: "...introduce a new static method that creates a random value."

File: CHANGELOG.md [hunk 5]
```diff
@@ -74,6 +74,51 @@ Swift 4.2
...
+  
+  ```swift
+  let diceRoll = Int.random(in: 1 ... 6)
+  let randomUnit = Double.random(in: 0 ..< 1)
+  let randomBool = Bool.random()
+  ```
+  
+  There are also a couple of collection additions included as well.
```
Comment (natecook1000) [inline CHANGELOG.md]: "There are also additions to select a random element from a collection or shuffle its contents."

**Review by natecook1000 — CHANGES_REQUESTED** [review]
Ah, thanks! Looks good, with some minor points here.

**Review by natecook1000 — APPROVED** [review]

**Comment by natecook1000** [comment]
LGTM, thanks @Azoy!

**Comment by natecook1000** [comment]
@swift-ci Please smoke test and merge

**Comment by natecook1000** [comment]
@swift-ci Please smoke test and merge

**Pull request merged**